docs/source/guides/ursula_configuration_guide.rst

```diff
@@ -81,34 +96,30 @@ If your installation is non-functional, be sure you have the latest version inst
 .. _Installation Guide: installation_guide.html
 
 
-
 3. Configure a new Ursula node
 --------------------------------
 
-With Docker
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Recall the volume mounts: `-v <path to a directory on your computer>:/root/.local/share/` which is where your Nucypher node stores persistent data as well as accesses ipc with your locally running geth node.
+Running an Ursula with Docker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Execute the following commands (Ubuntu):
+Assuming geth is running locally on goerli, Configure and run an Ursula using port and volume bindings:
 
 .. code:: bash
 
     export NUCYPHER_KEYRING_PASSWORD=<your keyring password>
-    export MY_IP=$(wget -q -O - ifconfig.me);
-    export NUCYPHER_WORKER_ADDRESS=<eth account checksum of your worker>
-    export NUCYPHER_STAKER_ADDRESS=<eth account checksum of your staker>
     export NUCYPHER_WORKER_ETH_PASSWORD=<your eth account password>
 
-    # init your worker
-    docker run -v /home/ubuntu:/root/.local/share/ -e NUCYPHER_KEYRING_PASSWORD -it nucypher/nucypher:latest nucypher ursula init --provider /root/.local/share/geth/.ethereum/goerli/geth.ipc --poa --worker-address $NUCYPHER_WORKER_ADDRESS --staker-address $NUCYPHER_STAKER_ADDRESS --rest-host $MY_IP
+    # Interactive Ursula-Worker Initialization
+    docker run -it -v ~/.ethereum:/root/.ethereum -v ~/.local/share/nucypher:/root/.local/share/nucypher -e NUCYPHER_KEYRING_PASSWORD nucypher:latest nucypher ursula init --provider file:///root/.ethereum/goerli/geth.ipc --staking-address <YOUR STAKING ADDRESS>
```

Comment:
staking vs. staker address?

Reply:
Given the command option uses 'staking' you are probably fine as is. That being said, maybe double-check if there is already a precedent in the docs, to be consistent.

Reply:
... but the command uses `staker`:
```python
    @click.option('--staker-address', help="Run on behalf of a specified staking account", type=EIP55_CHECKSUM_ADDRESS)
```

Reply:
:heavy_check_mark: